homeassistant/components/modbus/validators.py

```diff
@@ -104,6 +135,7 @@ def struct_validator(config: dict[str, Any]) -> dict[str, Any]:
         slave_count = config.get(CONF_VIRTUAL_COUNT, 0)
         slave_name = CONF_VIRTUAL_COUNT
     swap_type = config.get(CONF_SWAP, CONF_SWAP_NONE)
+    register_size_bytes = config.get(CONF_REGISTER_SIZE_BYTES, 2)
```

Comment:
Why have a default here ? the parameter already have a default.

Reply:
I'm using that because unit tests in test_init.py fail. Not totally sure but those tests don't seem to have the default set so they have a None value if i don't set it.
![imagem](https://github.com/home-assistant/core/assets/4754081/5fb5de4c-77ca-4fa5-b78d-9f959b8630eb)